snippets/olmv1-tp-extension-support.adoc

```diff
@@ -1,19 +1,20 @@
 // Text snippet included in the following modules:
 //
-// * modules/olmv1-installing-an-operator.adoc
+// * modules/olmv1-supported-extensions.adoc
 // * release_notes/ocp-4-16-release-notes.adoc (enteprise-4.16 branch only)
 // * release_notes/ocp-4-15-release-notes.adoc (enteprise-4.15 branch only)
 
 :_mod-docs-content-type: SNIPPET
 
-[IMPORTANT]
-====
-{olmv1} does not support dependency resolution. If an extension declares dependencies for other APIs or packages, the dependencies must be present on the cluster before you attempt to install the extension.
+Currently, {olmv1-first} supports installing cluster extensions that meet all of the following criteria:
 
-Currently, {olmv1} supports the installation of extensions that meet the following criteria:
-
-* The extension must use the `AllNamespaces` install mode.
+* The extension must use the `registry+v1` bundle format introduced in {olmv0}.
+* The extension must support installation via the `AllNamespaces` install mode.
 * The extension must not use webhooks.
+* The extension must not declare dependencies using any of the  following file-based catalog properties:
```

Comment:
```suggestion
* The extension must not declare dependencies that use any of the  following file-based catalog properties:
```

Reply:
From the IBM Style Guide: To avoid ambiguity, replace this gerund with either by using or that uses.